lib/mapnik_backend.js

```diff
@@ -108,6 +108,9 @@ MapnikSource.prototype._normalizeURI = function(uri) {
     // from cache even if they had a ttl set
     uri.query.metatileCache.deleteOnHit = uri.query.metatileCache.hasOwnProperty('deleteOnHit') ?
         as_bool(uri.query.metatileCache.deleteOnHit) : false;
+
+    if (typeof uri.query.mapnik_metrics === "undefined") uri.query.mapnik_metrics = false;
```

Comment:
Isn't `mapnik_` a bit redundant?

Reply:
Yup. This comes from a time I didn't have a lot of knowledge about the whole stack. I'll change it to just `metrics`.